tests/integration/test_celery.py

```diff
@@ -80,33 +77,20 @@ def expected_fluka_mock_results() -> dict:
     """Return expected result"""
     yield {
         "21": {
-            "4x4": [
-                [0., 0., 0., 0.],
-                [0., 0.00146468, 0.00169978, 0.],
-                [0.00090805, 0., 0., 0.],
-                [0, 0., 0., 0.]
-            ]
+            "4x4": [[0., 0., 0., 0.], [0., 0.00146468, 0.00169978, 0.], [0.00090805, 0., 0., 0.], [0, 0., 0., 0.]]
         },
         "22": {
-            "4x4": [
-                [0.00047575, 0., 0., 0.],
-                [0., 0.00188021, 0.00166488, 0.],
-                [0., 0.0010242, 0.00105254, 0.],
-                [0., 0., 0., 0.]
-            ]
+            "4x4": [[0.00047575, 0., 0., 0.], [0., 0.00188021, 0.00166488, 0.], [0., 0.0010242, 0.00105254, 0.],
+                    [0., 0., 0., 0.]]
         }
     }
 
 
 @pytest.mark.xfail(condition=platform.system() == "Windows", reason="FLUKA mock is not supported on Windows")
```

Comment:
Why do we use xfail here instead of skip?

Reply:
Good point, `skip` may be a better idea.  Good reasoning on the differences can be found here:
https://docs.pytest.org/en/6.2.x/skipping.html

Reply:
done